### DISCUSSION TITLE
Welcome to rlua Discussions!

### DISCUSSION THREAD

**Discussion by jugglerchris** [mlua-rs/rlua#222] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by greensourcecode** [comment]
Hello, noob here. Can tl (typed Lua) / Teal be used along side of this? Does this compile to Rust, is it written in Rust or both?

Thank you

**Reply by jugglerchris** [reply]
It looks like they're just implemented in plain Lua, so I see no reason it shouldn't work.

**Reply by greensourcecode** [reply]
Ah thank you. I'm playing with the idea of Teal with batteries included stuff (fun Lua, OOP stuff, etc) that compiles to Rust and Lua JIT. Maybe it's naiive? But either way this stuff is really interesting, I look forward to seeing where this goes.. if you guys want a website I'll gladly make it for you (it would be good for the portfolio), and pay for a domain name. Just let me me know

**Reply by jugglerchris** [reply]
Sounds like fun!
Thanks for the kind offer but we are not currently wanting a website.
